File: j2k/tests/testData/ast/kotlinApiAccess/ObjectMembers.java
//file
import kotlinApi.KotlinObject;

class C {
    int foo() {
        KotlinObject.INSTANCE$.setProperty1(1);
        KotlinObject.INSTANCE$.setProperty2(2);
        return KotlinObject.INSTANCE$.foo() +
               KotlinObject.INSTANCE$.getProperty1() +
               KotlinObject.INSTANCE$.getProperty2();
    }
}

File: kotlin-ide/j2k/new/testData/newJ2k/kotlinApiAccess/ObjectMembers.kt
import kotlinApi.KotlinObject
import kotlinApi.KotlinObject.property1
import kotlinApi.KotlinObject.property2

internal class C {
    fun foo(): Int {
        property1 = 1
        property2 = 2
        return KotlinObject.foo() +
                property1 +
                property2
    }
}